src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -408,6 +440,12 @@ private static DashboardStartupConfiguration LoadDashboardConfig(IConfiguration
             OtlpApiKey = otlpApiKey
         };
     }
+
+    private enum DashboardWebAppAuthMode
+    {
+        Unsecured,
```

Comment:
Unsecured vs None? I went with None but these should be consistent. Can you change the OTLP auth enum to use Unsecured. And update readme.

Reply:
Yes, I'll make a pass to get things consistent. We should also review the use of `A:B:C` vs `DOTNET_*` style config throughout the dashboard.